Tools/i18n/msgfmt.py

```diff
@@ -98,18 +102,49 @@ def generate():
     return output
 
 
-def make(filename, outfile):
-    ID = 1
-    STR = 2
-    CTXT = 3
+def make(filenames, outfile=None):
+    """ Compiles one or more po files(s).
 
+    filenames an iterable of strings representing the input file(s).
+    outfile is a string for the name of an output file or None.
+
+    If it is not None, the output file receives a merge of the input files.
+    If it is None, then for each file from filenames the name of the output
+    file is obtained by replacing the po extension with mo.
+    BEWARE: in previous version of make the first parameter was a string
+    containing a single filename.
```

Comment:
This note will soon became confusing, as it is not clear what version is previous. The right place for it is in the commit message, not in docstring.